target/linux/ipq40xx/files-4.14/arch/arm/boot/dts/qcom-ipq4019-eap2200.dts

```diff
@@ -0,0 +1,519 @@
+		};
+
+		qcom_crypto: qcrypto@8e20000 {
+			compatible = "qcom,qcrypto";
+			reg = <0x8e20000 0x20000>,
+				<0x8e04000 0x20000>;
+			reg-names = "crypto-base","crypto-bam-base";
+			interrupts = <0 207 0>;
+			qcom,bam-pipe-pair = <1>;
+			qcom,ce-hw-instance = <0>;
+			qcom,ce-hw-shared = <1>;
+			qcom,ce-device = <0>;
+			qcom,ce-opp-freq= <125000000>;
+			clocks = <&gcc GCC_CRYPTO_CLK>,
+				<&gcc GCC_CRYPTO_AXI_CLK>,
+				<&gcc GCC_CRYPTO_AHB_CLK>;
+
+			clock-names = "core_clk", "bus_clk", "iface_clk";
+			status = "ok";
+		};
+
+		qcom_cedev: qcedev@8e20000 {
+			compatible = "qcom,qcedev";
+			reg = <0x8e20000  0x20000>,
+				<0x8e04000  0x20000>;
+			reg-names = "crypto-base","crypto-bam-base";
+			interrupts = <0 207 0>;
+			qcom,bam-pipe-pair = <1>;
+			qcom,ce-hw-instance = <0>;
+			qcom,ce-hw-shared = <1>;
+			qcom,ce-device = <0>;
+			qcom,ce-opp-freq= <125000000>;
+			clocks = <&gcc GCC_CRYPTO_CLK>,
+				<&gcc GCC_CRYPTO_AXI_CLK>,
+				<&gcc GCC_CRYPTO_AHB_CLK>;
+
+			clock-names = "core_clk", "bus_clk", "iface_clk";
+			status = "ok";
+		};
+		wifi_glb_tcsr: tcsr@1949000 {
+		        compatible = "qcom,tcsr";
+		        reg = <0x1949000 0x100>;
+		        qcom,wifi_glb_cfg = <TCSR_WIFI_GLB_CFG>;
+		};
+
+		wifi_noc_memtype_m0_m2_tcsr: tcsr@1957000 {
+		        compatible = "qcom,tcsr";
+		        reg = <0x1957000 0x100>;
+			qcom,wifi_noc_memtype_m0_m2 = <TCSR_WIFI_NOC_MEMTYPE_M0_M2>;
+		};
+		tcsr: tcsr@194b000 {
+		        compatible = "qcom,tcsr";
+		        reg = <0x194b000 0x100>;
+		        qcom,usb-hsphy-mode-select = <TCSR_USB_HSPHY_HOST_MODE>;
+		        status = "disabled";
+		};
+
+		ess_tcsr: ess_tcsr@1953000 {
+		        compatible = "qcom,tcsr";
+		        reg = <0x1953000 0x1000>;
+		        qcom,ess-interface-select = <TCSR_ESS_PSGMII>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+		button@1 {
+                        label = "wps";
+                        linux,code = <KEY_WPS_BUTTON>;
+                        gpios = <&tlmm 18 GPIO_ACTIVE_LOW>;
+			linux,input-type = <1>;
+                };
+	};
+
+	aliases {
+		led-boot = &power;
+		led-failsafe = &power;
+		led-running = &power;
+		led-upgrade = &power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+                        power: orange {
+                                label = "eap2200:orange:power";
+                                gpios = <&tlmm 43 GPIO_ACTIVE_LOW>;
+                        };
+
+                        lan1 {
+                                label = "eap2200:blue:lan1";
+                                gpios = <&tlmm 44 GPIO_ACTIVE_LOW>;
+                        };
+
+                        lan2 {
+                                label = "eap2200:blue:lan2";
+                                gpios = <&tlmm 45 GPIO_ACTIVE_LOW>;
+                        };
+
+                        wlan2g {
+                                label = "eap2200:blue:wlan2g";
+                                gpios = <&tlmm 46 GPIO_ACTIVE_LOW>;
```

Comment:
Perhaps `linux,default-trigger` can be used here and below instead of `01_leds` for WLAN LEDs?
e.g. `linux,default-trigger = "phy0tpt";`

Reply:
Thanks for advice.